src/reducers/sources.js

```diff
@@ -431,7 +431,7 @@ function getSourcesByUrlInSources(
   return urls[url].map(id => sources[id]);
 }
 
-export function getSourceInSources(sources: SourcesMap, id: string): ?Source {
+export function getSourceInSources(sources: SourcesMap, id: string): Source {
```

Comment:
I don't think this is safe. If the `id` doesn't exist in the map, we'll get `undefined` as the result. If we _did_ want this type, I think we'd want this to be

```
const source = sources[id];
if (!source) {
  throw new Error(`Unknown source ${id}`);
}
return source;
```
but looking at places like https://github.com/devtools-html/debugger.html/blob/cbdcc6f424e13c6e919288c31235675f818c8937/src/actions/project-text-search.js#L74 there appear to be places where we explicitly handle the fact that the source might not exist. 

Given that, it might be better to leave this as `?Source` for now and handle it explicitly in `getMappedLocation` by throwing there instead, because otherwise I'd want to do a pass over the whole codebase trying to figure out if we can get away throwing in `getSourceInSources`.